specification/introduction/index.md

```diff
@@ -2,7 +2,7 @@
 title: Introduction
 ---
 # 1 Introduction
-This document introduces the concept of a Common Specification for Information Packages. It aims to serve three main purposes:
+This document introduces the concept of the Common Specification for Information Packages (CS IP). It aims to serve three main purposes:
```

Comment:
I think it might be better if we adopt CSIP as the acronym rather than add a space?